src/test/overlay/short_read_test.cpp

```diff
@@ -68,7 +68,7 @@ class short_read_test : public beast::unit_test::suite
     endpoint_type
     endpoint()
     {
-        return endpoint_type(address_type::from_string("127.0.0.1"), 9000);
+        return endpoint_type(address_type::from_string("127.0.0.1"), 0);
```

Comment:
thanks for pointing that out - that doesn't seem like what we want and I clearly need to investigate more